packages/app-project/src/helpers/getStaticPageProps/getStaticPageProps.js

```diff
@@ -17,10 +17,10 @@ export default async function getStaticPageProps({ params, query }) {
   if (params.owner && params.project) {
     const projectSlug = `${params.owner}/${params.project}`
     const project = await fetchProjectData(projectSlug, { env })
-    if (!project.id) {
+    applySnapshot(store.project, project)
+    if (!store.project.id) {
       return notFoundError(`Project ${params.owner}/${params.project} was not found`)
     }
-    applySnapshot(store.project, project)
```

Comment:
@goplayoutside3 this was the source of the `cannot read property length of undefined` error that you saw yesterday. `applySnapshot` failed here, because of the null researcher quote, so the project has no active workflows later, when this function tries to fetch linked project workflows.

I've restored the check that we had previously, which checked if the project model has an ID, rather than the returned data from the API helper.